api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -268,6 +269,12 @@ public long getQueryBatchSize() {
                 .creationTime(offsetDateTimeUtc(rs.getTimestamp("creation_time")))
                 .currentPosition(rs.getString("current_position"))
                 .dataAccessLevel(dataAccessLevelFromStorage(rs.getShort("data_access_level")))
+                // TODO placeholder until we have a proper association of users to tiers
+                .accessTierShortNames(
```

Comment:
Now we will have `dataAccessLevel` and `accessTierShortNames` showing the same data? Or `dataAccessLevel` will be replaced by the new thing in the future?

Reply:
dataAccessLevel is the deprecated version.  This is one step in the process to replace it with accessTier.